external/webserver/http_client.c

```diff
@@ -237,7 +263,9 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 					}
 					if (strcmp(key, "Content-Length") == 0) {
 						len->content_len = HTTP_ATOI(value);
-						response->total_len = len->content_len;
+						if(response){
+							response->total_len = len->content_len;
```

Comment:
Is it okay not to check `len` is NULL or not before using?

Reply:
len is pointer to local structure in http_recv_and_handle_request so it will not be NULL.